cfme/fixtures/v2v.py

```diff
@@ -14,9 +15,12 @@
 @pytest.fixture(scope='function')
 def v2v_providers(request, second_provider, provider):
     """ Fixture to setup providers """
+    v2v_providers = namedtuple(
+        'v2v_providers', ['vmware_provider', 'rhv_provider']
+    )
     setup_or_skip(request, second_provider)
     setup_or_skip(request, provider)
-    yield second_provider, provider
+    yield v2v_providers(vmware_provider=second_provider, rhv_provider=provider)
```

Comment:
**Required** I think we are still making the assumption that the rhv provider will always be provider and vmware will always be the second provider, I think we should move away from this assumption and specifically check.

Reply:
will check that here.